packages/Webkul/Velocity/src/Resources/views/admin/content/edit.blade.php

```diff
@@ -7,7 +7,8 @@
 @section('content')
     <div class="content">
         @php
-            $locale = request()->get('locale') ?: app()->getLocale();
+            $locale = request()->get('locale') ?: 'en';
```

Comment:
The locale should not be a static value.

Reply:
Dynamic Added